samples/csharp/MicrosoftGraphSDK/Submission.cs

```diff
@@ -157,5 +188,106 @@ public class Submission
                 throw new GraphException($"ReassignAsync call: {ex.Message}", ex, classId, assignmentId, submissionId);
             }
         }
+
+        /// <summary>
+        /// Creating Feedback resource outcome
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <param name="submissionId">Student submission id</param>
+        /// <returns>EducationSubmission</returns>
+        public static async Task<EducationOutcome> CreateFeedbackResourceOutcome(
+    GraphServiceClient client,
+    string classId,
+    string assignmentId,
+    string submissionId)
+        {
+            var educationOutcome = new EducationFeedbackResourceOutcome
+            {
+                OdataType = "#microsoft.graph.educationFeedbackResourceOutcome",
+                FeedbackResource = new EducationWordResource
+                {
+                    OdataType = "#microsoft.graph.educationWordResource",
+                    DisplayName = "Document1.docx"
+                }
+            };
+
+            return await client.Education
+                .Classes[classId]
+                .Assignments[assignmentId]
+                .Submissions[submissionId]
+                .Outcomes
+                .PostAsync(educationOutcome);
+        }
+
+        /// <summary>
+        /// Creates a feedback resource for a given submission
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <param name="submissionId">Student submission id</param>
+        /// <param name="educationOutcome">Education outcome object</param>
```

Comment:
method and parameters description don't reflect what the method does, please check and adjust

Reply:
udpated to "        /// <summary>
        /// Update the properties of an educationOutcome object
        /// </summary>
        /// <param name="client">Microsoft Graph service client</param>
        /// <param name="classId">User class id</param>
        /// <param name="assignmentId">Assignment id in the class</param>
        /// <param name="submissionId">Student submission id</param>
        /// <param name="outcomeId">Outcome id</param>
        /// <param name="educationOutcome">Education outcome object</param>
        /// <returns>EducationOutcome</returns>"